.php_cs

```diff
@@ -19,6 +19,13 @@ return PhpCsFixer\Config::create()
         'space_after_semicolon' => false,
         'yoda_style' => false,
         'no_break_comment' => false,
+        'native_function_invocation' => false,
+        'native_constant_invocation' => false,
+        'phpdoc_types_order' => false,
+        'php_unit_mock_short_will_return' => false,
+        'php_unit_construct' => false,
+        'standardize_increment' => false,
+        'fopen_flags' => false,
```

Comment:
So we deal with taking advantage of improved CS fixer to improving our CS as a followup then?

I guess that is ok'ish, there are some other rules to consider as well: https://github.com/ezsystems/ezplatform-xmltext-fieldtype/blob/master/.php_cs#L22-L26

Reply:
> So we deal with taking advantage of improved CS fixer to improving our CS as a followup then?
> 
> I guess that is ok'ish, there are some other rules to consider as well: https://github.com/ezsystems/ezplatform-xmltext-fieldtype/blob/master/.php_cs#L22-L26

Yep, kept for follow-up because diff was too big. Could also take in these additional ones.